CONTRIBUTING.md

```diff
@@ -1,4 +1,28 @@
 Contributing
 ============
 
-ADD ACTUAL CONTRIBUTING GUIDLINES
+Thanks for helping out!
```

Comment:
:+1: Awesome, just opened an issue to track it https://github.com/FormidableLabs/generator-formidable-react-component/issues/24